operations/cluster.md

```
@@ -34,89 +34,81 @@ Next, you need to launch multiple FalkorDB instances that will form the cluster.
+
+```bash
+redis-cli -c
+```
+B):
+
+```bash
+GRAPH.QUERY testGraph "UNWIND range(1, 100) AS id CREATE (n:Person {id: id, name: 'Person ' + toString(id), age: 20 + id % 50})"
+```
+The out put will show that it was moved to the right node:
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)